Lib/fontTools/cffLib.py

```diff
@@ -1731,17 +1743,16 @@ def xmlRead(self, name, attrs, content, parent):
 	((12, 36),	'FDArray',		'number',	None,	FDArrayConverter()),
 	(17,		'CharStrings',		'number',	None,	CharStringsConverter()),
 	(24,		'VarStore',		'number',	None,	VarStoreConverter()),
-	(25,		'maxstack',		'number',	None,	None),
```

Comment:
Yes.  maxstack needs to be set before any PrivateDict is processed, in order to be able to parse the blend operators correctly. The simplest way to do this is to put maxstack first in the order, although it only needs to be before the FDArray.